pkg/download/goget/goget_test.go

```diff
@@ -0,0 +1,37 @@
+package goget
+
+import (
+	"context"
+	"testing"
+)
+
+type testCase struct {
+	name    string
+	mod     string
+	version string
+}
+
+// TODO(marwan): we should create Test Repos under github.com/gomods
+// so we can get reproducible results from live VCS repos.
+// For now, I cannot test that github.com/pkg/errors returns v0.8.0
+// from goget.Latest, because they could very well introduce a new tag
+// in the near future.
```

Comment:
sounds like a great idea! mind creating an issue for it?

Reply:
yep

Reply:
Can we update the comment to ref: https://github.com/gomods/athens/issues/340